src/Sulu/Bundle/AdminBundle/Resources/js/components/CircleSelection/CircleSelection.js

```diff
@@ -0,0 +1,88 @@
+// @flow
+import type {Node} from 'react';
+import React from 'react';
+import withContainerSize from '../withContainerSize';
+import CircleSelectionRenderer from './CircleSelectionRenderer';
+import type {SelectionData} from './types';
+import circleSelectionStyles from './circleSelection.scss';
+
+type Props = {
+    children?: Node,
+    disabled: boolean,
+    filled: boolean,
+    label?: string,
+    maxRadius?: number,
+    minRadius?: number,
+    onChange: (value: ?SelectionData) => void,
+    percentageValues: boolean,
+    resizable: boolean,
+    round: boolean,
+    value: SelectionData | typeof undefined,
+};
+
+class CircleSelection extends React.Component<Props & {
+    containerHeight: number,
+    containerWidth: number,
+}> {
+    static defaultProps = {
+        disabled: false,
+        filled: false,
+        percentageValues: false,
+        resizable: true,
+        round: true,
+    };
+
+    render() {
+        const {
+            children,
+            containerHeight,
+            containerWidth,
+            disabled,
+            filled,
+            label,
+            maxRadius,
+            minRadius,
+            onChange,
+            percentageValues,
+            resizable,
+            round,
+            value,
+        } = this.props;
+
+        return (
+            <div className={circleSelectionStyles.selection}>
+                {children}
+                <CircleSelectionRenderer
+                    containerHeight={containerHeight}
+                    containerWidth={containerWidth}
+                    disabled={disabled}
+                    filled={filled}
+                    label={label}
+                    maxRadius={maxRadius}
+                    minRadius={minRadius}
+                    onChange={onChange}
+                    percentageValues={percentageValues}
+                    resizable={resizable}
+                    round={round}
+                    value={value}
+                />
+            </div>
+        );
+    }
+}
+
+export {
+    CircleSelection,
```

Comment:
For the `CircleSelection` it wouldn't be necessary, but we need that for the `RectangleSelection` because of the `ImageRectangleSelection` and I wanted to keep these consistent.

Reply:
Just had a look, I think it was not necessary for the `ImageRectangleSelection`... The `RectangleSelection` export only seems to be introduced for the tests :confused: Maybe the tests were a bit easier to write this way... I would prefer in both cases not to have them, but I don't want to make you refactor the `Rectangle` test in this PR as well... But could we agree to remove this export in here, if there is no need for it? In case it is needed at some point we can still introduce it later. If we introduce it now we can't remove it anywhen in the future, because it would be a BC break.

Reply:
I've seen that you've implemented the tests in the same way, so it is ok for me to keep it currently as it is.